tests/nfc/nfc_event_getter_test.c

```diff
@@ -0,0 +1,55 @@
+#include "board.h"
+#include "buzzer.h"
+#include "events.h"
+#include "nfc.h"
+#include "nfc_events.h"
+#include "ui_instruction.h"
+
+#ifdef NFC_EVENT_CARD_DETECT_MANUAL_TEST
+
+void test_nfc_operation_with_os_getter() {
```

Comment:
Let's add this test in `unit_tests_list.c` under the same compilation macro `NFC_EVENT_CARD_DETECT_MANUAL_TEST`. We should add a brief comment there as to why that test is added in a special way.

Reply:
I feel this could be called in unit_test_main.c rather than unit_test_list.c as the test is of manual category, otherwise another test could be waiting for this test to complete when enabled.

Reply:
No, we should not create exception in the way we are adding tests.
We should create this test in the same way we create other tests, only keep it under compilation macro guard.

So, please create a test group etc, as done in other modules.